addon/components/modal-dialog/component.js

```diff
@@ -0,0 +1,8 @@
+import ModalDialog from 'ember-modal-dialog/components/modal-dialog';
+import config from 'dummy/config/environment';
+
+export default ModalDialog.extend({
+
+  renderInPlace: config.environment === 'test'
```

Comment:
Since other parts of the code access the environment config through the Owner API, this probably should too.

Reply:
yep, I agree we should have consistent style.

Can someone explain to me the difference between importing config and

```js
let config = getOwner(this).resolveRegistration('config:environment');
```

?

Reply:
To import config, you have to know the name of the host app. Since we're always targeting addons' dummy apps, we _could_ get away with assuming `dummy` in the import path, but there's nothing stopping someone from picking a different `modulePrefix` for their dummy app, so using the owner API is safer.